README.md

```diff
@@ -13,106 +12,135 @@ Cert-issuer v3 is _not_ backwards compatible and does not support Blockcerts v2
+   ```
+   issuer=`bitcoin-cli getnewaddress`
+   sed -i.bak "s/<issuing-address>/$issuer/g" /etc/cert-issuer/conf.ini
+   bitcoin-cli dumpprivkey $issuer > /etc/cert-issuer/pk_issuer.txt
+   ```
+
+   `sed` command allows us to quickly remove or replace the content without having to open a file.
+
+4. Don't forget to save snapshots so you don't lose your work (see step 4 of client setup)
 
 ## Issuing certificates
 
-1. Add your certificate to /etc/cert-issuer/data/unsigned_certificates/.
+1.  Add your certificate to `/etc/cert-issuer/data/unsigned_certificates/`:
 
     ```
     # To use a sample unsigned certificate as follows:
-    cp /cert-issuer/examples/data-testnet/unsigned_certificates/3bc1a96a-3501-46ed-8f75-49612bbac257.json /etc/cert-issuer/data/unsigned_certificates/
+    cp /cert-issuer/examples/data-testnet/unsigned_certificates/verifiable-credential.json /etc/cert-issuer/data/unsigned_certificates/
 
     # If you created your own unsigned certificate using cert-tools (assuming you placed it under data/unsigned_certificates):
     cp <cert-issuer-home>/data/unsigned_certificates/<your-cert-guid>.json /etc/cert-issuer/data/unsigned_certificates/
     ```
 
-2. Make sure you have enough BTC in your issuing address.
-
-    a. You're using bitcoind in regtest mode, so you can print money. This should give you 50 (fake) BTC:
+2.  Make sure you have enough BTC in your issuing address. You're using bitcoind in regtest mode, so you can print money. This should give you 50 (fake) BTC:
 
     ```
-    bitcoin-cli generate 101
+    bitcoin-cli -generate 101
     bitcoin-cli getbalance
     ```
 
-    b. Send the money to your issuing address -- note that bitcoin-cli's standard denomination is bitcoins not satoshis! (In our
-    app, the standard unit is satoshis.) This command sends 5 bitcoins to the address
+3.  Allow fallback fee in your `bitcoin.conf` (use Vim as the text editor). Add this line to `bitcoin.conf`: `fallbackfee=0.00001`
+
+    ```
+    vi /root/.bitcoin/bitcoin.conf
+    ```
+
+    You might have to kill bitcoind daemon using `ps aux` and `kill <pid>` , then start it again with `bitcoind -daemon=<path to bitcoin conf file>`
+
+4.  Send the money to your issuing address. Note that bitcoin-cli's standard denomination is bitcoins not satoshis! (In our app, the standard unit is satoshis.) This command sends 5 bitcoins to the address:
 
     ```
     bitcoin-cli sendtoaddress $issuer 5
     ```
 
-3. Issue the certificates on the blockchain
+5.  Issue the certificates on the blockchain:
 
     ```
     cert-issuer -c /etc/cert-issuer/conf.ini
     ```
 
-4. Your Blockchain certificates are located in `/etc/cert-issuer/data/blockchain_certificates`. Copy these to your local machine, and add them to cert-viewer's `cert_data` folder to see your certificates in the Certificate Viewer.
+    **Optional:** add `--verification_method` with issuer's DID (learn more about Decentralized Identifiers [here](https://www.w3.org/TR/did-core/))
```

Comment:
I would argue that this is not optional.
Technically we don't _yet_ make verification of keys if we are pointing to a URL and not a DID, but a URL for verification method is valid. Could you please also add a link to this section? https://github.com/blockchain-certificates/cert-issuer/#working-with-dids
It is good practice though to maintain the verification method. I am even wondering if the issuance is not failing if not specified?